### PR TITLE
[MB-8981] Billable weight card component

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Billable weight card component is added to storybook for future use

## Reviewer Notes
- For some of the padding/margin values in the Zeplin design I rounded/down to be consistent with uswds spacing values being divisible by 8

## Setup
```sh
make storybook
```
Search for component named `BillableWeightCard`

## Code Review Verification Steps
* [x] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-8981) for this change

## Screenshots
Screen Shot 2021-08-12 at 12 00 38 AM

### CODE REVIEW

**Review by pearl-truss — APPROVED** [review]
LGTM

**Review by Shkeating — APPROVED** [review]
When I looked at the Happo added images, I noticed the safari one was WILDIN:
image

So I tested it locally, and it looks fine?
image

I'm going to approve, but maybe try getting Happo to re run? 🤷‍♀️

File: src/components/Office/BillableWeightCard/BillableWeightCard.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <div className={classnames(styles.cardContainer, 'container')}>
+      <div className={styles.cardHeader}>
+        <h2>Billable weights</h2>
+        <Button>Review weights</Button>
+      </div>
+      <div className={styles.spaceBetween}>
+        <div>
+          <h5>Maximum billable weight</h5>
+          <h4>{maxBillableWeight} lbs</h4>
```
Comment (YanZ777) [inline src/components/Office/BillableWeightCard/BillableWeightCard.jsx:23]: Minor thing about the props:
we should expect the component to format the weight itself (`import { formatWeight } from 'shared/formatters';`)
in which case the weights should be passed in as ints
or
we should expect that whomever uses this component formats the weights
in which case the weights will be strings with the lbs already

Comment (Ronolibert) [reply src/components/Office/BillableWeightCard/BillableWeightCard.jsx:23]: Gonna use that! Didn't know that existed

**Pull request merged**